model-optimizer/extensions/ops/ReduceOps.py

```diff
@@ -24,6 +24,7 @@
 reduce_map = {
     'ReduceSum': np.sum,
     'ReduceProd': np.prod,
+    'ReduceL2': lambda x, axis, keepdims: np.sqrt(np.sum(a=np.square(x), axis=axis, keepdims=keepdims)),
```

Comment:
can we use numpy.linalg.norm? https://numpy.org/doc/1.18/reference/generated/numpy.linalg.norm.html

Reply:
No, at least because it supports only 1D and 2D shapes.